samples/openthread/cli/sample.yaml

```diff
@@ -1,77 +1,113 @@
 sample:
   name: OpenThread CLI sample
   description: Test OpenThread Command Line Interface.
+
 tests:
-  sample.openthread.cli:
+  sample.openthread.cli.multiprotocol:
     build_only: true
-    platform_allow: >
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840
-      nrf52840dongle_nrf52840 nrf21540dk_nrf52840
     tags: ci_build
+    platform_allow: >
+      nrf52840dk_nrf52840
+      nrf21540dk_nrf52840
+      nrf5340dk_nrf5340_cpuapp
+      nrf5340dk_nrf5340_cpuapp_ns
     extra_args: >
-      SNIPPET="ci;multiprotocol;tcp"
+      SNIPPET="ci;logging;multiprotocol"
     integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf21540dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
       - nrf5340dk_nrf5340_cpuapp_ns
+
+  sample.openthread.cli.singleprotocol:
+    build_only: true
+    tags: ci_build
+    platform_allow: >
+      nrf52840dk_nrf52840
+      nrf21540dk_nrf52840
+      nrf5340dk_nrf5340_cpuapp
+      nrf5340dk_nrf5340_cpuapp_ns
+    extra_args: >
+      SNIPPET="ci;logging"
+    integration_platforms:
       - nrf52840dk_nrf52840
-      - nrf52840dongle_nrf52840
       - nrf21540dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+      - nrf5340dk_nrf5340_cpuapp_ns
+
   sample.openthread.cli.thread_1_1:
     build_only: true
-    platform_allow: >
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf21540dk_nrf52840
     tags: ci_build
+    platform_allow: >
+      nrf52840dk_nrf52840
+      nrf5340dk_nrf5340_cpuapp
     extra_args: >
-      CONFIG_OPENTHREAD_THREAD_VERSION_1_1=y
       SNIPPET=ci
+      CONFIG_OPENTHREAD_THREAD_VERSION_1_1=y
     integration_platforms:
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf5340dk_nrf5340_cpuapp_ns
       - nrf52840dk_nrf52840
-      - nrf21540dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+
   sample.openthread.cli.usb:
     build_only: true
-    platform_allow: >
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf21540dk_nrf52840
     tags: ci_build
+    platform_allow: >
+      nrf52840dk_nrf52840
     extra_args: >
-      SNIPPET="usb;ci;multiprotocol;tcp"
+      SNIPPET="ci;logging;multiprotocol;usb"
     integration_platforms:
-      - nrf5340dk_nrf5340_cpuapp
```

Comment:
usb build targets should be restored as well

Reply:
restored